src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -369,6 +378,16 @@ public InsertValidationResponse insertRows(
     return response;
   }
 
+  /**
+   * Insert a batch of rows into the channel with the end offset token only, please see {@link
+   * SnowflakeStreamingIngestChannel#insertRows(Iterable, String, String)} for more information.
+   */
+  @Override
+  public InsertValidationResponse insertRows(
+      Iterable<Map<String, Object>> rows, String offsetToken) {
+    return insertRows(rows, null, offsetToken);
```

Comment:
When is this invoked.

Reply:
I think the github diff is making things confusing, this is the old interface that customer currently used to insert rows with the rows and the end_offset_token, the new interface added is to allow them pass start_offset_token as well